src/main/java/archibald/Archibald.java

```diff
@@ -0,0 +1,206 @@
+package archibald;
+import java.util.Scanner;
+import java.io.*;
```

Comment:
You should import the libraries individually instead of using a wildcard (i.e. *).